init.fish

```diff
@@ -10,9 +10,10 @@ abbr -a g          git
 abbr -a ga         git add
 abbr -a gaa        git add --all
 abbr -a gapa       git add --patch
+abbr -a gb         git branch -vv
 abbr -a gba        git branch -a -v
 abbr -a gban       git branch -a -v --no-merged
-abbr -a gb         git branch -vv
+abbr -a gbage      "git for-each-ref --sort=committerdate refs/heads/ --format=\"%(HEAD) %(color:yellow)%(refname:short)%(color:reset) - %(color:red)%(objectname:short)%(color:reset) - %(contents:subject) - %(authorname) (%(color:green)%(committerdate:relative)%(color:reset))\""
```

Comment:
Abbreviations are best for shorter commands that one might edit or append to after typing.  This looks like it belongs in a function to me.

Reply:
Of course. Fixed!